tests/dodo.py

```diff
@@ -15,7 +15,7 @@
 __version__ = "0.1"
 
 __maintainer__ = "Daren Thomas"
-__email__ = "thomas@arch.ethz.ch"
+__email__ = "cea@arch.ethz.ch"
```

Comment:
a change like this, applied to all the files in the repository, should probably be done in a separate branch with an issue to do it.